README.md

```diff
@@ -242,6 +291,9 @@ Please be aware of the following:
   If you must add a default, for example for instantiating your dataclass elsewhere, do `config: Config = field(default_factory=Config)`, assuming that all fields in `Config` have a default.
 - Nested dataclasses can not be positional (although _fields of_ the nested dataclass can be).
 - Argument names must not collide. In the example above, the `Base` class should not contain a field named `config_field_a`.
+- When reading [defaults from a file](#loading-defaults-from-file), the data inside the file may be nested like the dataclasses as well as flat with prefixes.
+  E.g. `{"config": {"field_b": "xyz"}` has the same effect as `{"config_field_b": "xyz"}`. Pydargs will raise an
+  exception in the case of collisions between these alternatives.
```

Comment:
It's unclear to me when the exception will raise

1) When using different formats, regardless of whether they are about the same field
2) When using different formats, when they are modifying the same field

Reply:
Would this be better?
```suggestion
  exception in the case of collisions between keys in alternative formats.
```

Reply:
Yes, better!